### PR TITLE
unbacktick internal names

### LINKED ISSUE #225: Remove some double quotes of defined names
At least the ones in upper case. Or maybe it's the backtick to RFC text translation that does that ?

As reported by Robert Sparks
https://mailarchive.ietf.org/arch/msg/cellar/gCFVwpnS79yZFsGk6yXucB5sMn0

### PR DESCRIPTION
**Pull request by robUx4**
It doesn't produce readable RFC text (or we would need to customize xml2rfc to
keep a nice emphasis in the HTML version).

Fixes #225

### CODE REVIEW

**Comment by robUx4** [comment]
The backtick is kept for XML elements in the code. The double quoting in the TEXT output is actually welcome.

File: specification.markdown [hunk 1]
```diff
@@ -1,100 +1,100 @@
 # Introduction
 
-`EBML`, short for Extensible Binary Meta Language, specifies a binary and octet (byte) aligned format inspired by the principle of XML (a framework for structuring data).
+EBML, short for Extensible Binary Meta Language, specifies a binary and octet (byte) aligned format inspired by the principle of XML (a framework for structuring data).
```
Comment (retokromer) [inline specification.markdown]: Should the first occurrence not be «quoted», as definition?

Comment (robUx4) [reply specification.markdown]: Fixed in the next commits

Comment (robUx4) [reply specification.markdown]: In the `Notation and Conventions` section, not the main text.

File: specification.markdown [hunk 2]
```diff
@@ -160,7 +160,7 @@ Octet Length | Possible Value Range
 7            | 0 to 2^49-2
 8            | 0 to 2^56-2
 
-If the length of `Element Data` equals `2^(n*7)-1` then the octet length of the `Element Data Size` MUST be at least `n+1`. This rule prevents an `Element Data Size` from being expressed as a reserved value. The following table clarifies this rule by showing a valid and invalid expression of an `Element Data Size` with a `VINT_DATA` of 127 (which is equal to 2^(1*7)-1) and 16,383 (which is equal to 2^(2*7)-1).
+If the length of Element Data equals `2^(n*7)-1 then the octet length of the Element Data Size MUST be at least n+1`. This rule prevents an Element Data Size from being expressed as a reserved value. The following table clarifies this rule by showing a valid and invalid expression of an Element Data Size with a VINT_DATA of 127 (which is equal to 2^(1*7)-1) and 16,383 (which is equal to 2^(2*7)-1).
```
Comment (mkver) [inline specification.markdown]: The backticked block here includes two forumulae and the statement in between.

Comment (robUx4) [reply specification.markdown]: OK I will merge your patch and then update this one.

**Comment by dericed** [comment]
@robUx4 I'll try to resolve this one for re-review.

File: specification.markdown (hunk 2 above)
Comment (dericed) [reply specification.markdown]: There's still two backticks left in here which join together two formula and text into a part that shouldn't be singularly encapsulated.

File: specification.markdown [hunk 3]
```diff
@@ -271,112 +271,112 @@ After  | 0x3B4040   | 0x82              | 0x6869       | 0xEC80
...
 
-An `EBML Schema` MUST declare exactly one `EBML Element` at `Root Level` (referred to as the `Root Element`) that occurs exactly once within an `EBML Document`. The `Void Element` MAY also occur at `Root Level` but is not a `Root Element` (see [the definition of the `Void Element`](#void-element)).
+An EBML Schema MUST declare exactly one EBML Element at Root Level (referred to as the Root Element) that occurs exactly once within an EBML Document. The Void Element MAY also occur at Root Level but is not a Root Element (see [the definition of the Void Element](#void-element)).
 
-The `EBML Schema` MUST document all Elements of the `EBML Body`. The `EBML Schema` does not document `Global Elements` that are defined by this document (namely the `Void Element` and the `CRC-32 Element`).
+The EBML Schema MUST document all Elements of the EBML Body. The EBML Schema does not document Global Elements that are defined by this document (namely the Void Element and the CRC-32 Element).
 
 The EBML Schema` MUST NOT use the Element ID `0x1A45DFA3` which is reserved for the EBML Header for resynchronization purpose.
```
Comment (dericed) [inline specification.markdown]: oops, stray backtick in this line

Comment (robUx4) [reply specification.markdown]: Fixed in the next version

File: specification.markdown [hunk 4]
```diff
@@ -399,162 +399,162 @@ PathMinOccurrence        = 1*DIGIT
...
-A `lang` attribute which is set to the [@!RFC5646] value of the language of the element's documentation.
+A lang attribute which is set to the [@!RFC5646] value of the language of the element's documentation.
 
-The `lang` attribute is OPTIONAL.
+The lang attribute is OPTIONAL.
 
 #### purpose
 
-A `purpose` attribute distinguishes the meaning of the documentation. Values for the `<documentation>` sub-element's `purpose` attribute MUST include one of the following: `definition`, `rationale`, `usage notes`, and `references`.
+A purpose attribute distinguishes the meaning of the documentation. Values for the <documentation> sub-element's purpose attribute MUST include one of the following: 'definition`, 'rationale`, 'usage notes`, and 'references`.
```
[the remark this answers is not in the document]
Comment (robUx4) [reply specification.markdown]: My mistake, fixed more generally with elements that are found in XML

**Comment by robUx4** [comment]
Can we agree on merging this ? Because it's a PITA to maintain and keep writing new patches in a form that will need to be edited.

File: specification.markdown (hunk 2 above)
Comment (robUx4) [reply specification.markdown]: Fixed

**Comment by dericed** [comment]
agreed

**Comment by dericed** [comment]
fyi, after merging this, I bumped the version 10 merged to follow it (rather the precede)

**Pull request merged**